microprofile/testing/junit5/src/main/java/io/helidon/microprofile/testing/junit5/HelidonJunitExtension.java

```diff
@@ -569,6 +613,25 @@ void registerOtherBeans(@Observes AfterBeanDiscovery event) {
                     .scope(ApplicationScoped.class)
                     .createWith(context -> getWebTarget(client, "@default"));
 
+            // Register all mocks
+            mocks.forEach(type -> {
+                event.addBean()
+                    .addType(type)
+                    .scope(ApplicationScoped.class)
+                    .alternative(true)
+                    .createWith(inst -> {
+                        Set<Bean<?>> beans = beanManager.getBeans(MockSettings.class);
```

Comment:
This is the code that would fail at runtime, if you use `@MockBean` and you do not have mockito on the classpath. We should never do this.
So either we need extensibility and have a mockito support for `MockBean`, or fallback to some other approach (such as creating a no-op proxy) if it is not available.